packages/plugin-css/test/border/want.css

```diff
@@ -22,17 +22,17 @@
 
 @supports (color: color(display-p3 1 1 1)) {
   :root {
-    --ds-color-gray: color(display-p3 0.329412 0.278431 0.254902);
-    --ds-border: 1px solid color(display-p3 0.05098 0.011765 0);
+    --ds-color-gray: color(display-p3 0.32941176470588235 0.2784313725490196 0.2549019607843137);
```

Comment:
🤔 I guess all the rounding I was doing was slowing my library down for no good reason